.github/workflows/qa.yml

```diff
@@ -92,13 +92,15 @@ jobs:
       url: https://qa.applicant-portal.planninglabs.nyc
     runs-on: ubuntu-latest
     env:
+      FEATURE_FLAG_EXCLUDE_FROM_SEARCH_RESULTS: ${{vars.FEATURE_FLAG_EXCLUDE_FROM_SEARCH_RESULTS}}
       FEATURE_FLAG_SELF_SERVICE: ${{ vars.FEATURE_FLAG_SELF_SERVICE }}
       FEATURE_FLAG_PHISHING_WARNING: ${{ vars.FEATURE_FLAG_PHISHING_WARNING }}
       HOST: ${{ vars.ZAP_API_HOST }}
       MAINTENANCE_START: ${{ vars.MAINTENANCE_START }}
       MAINTENANCE_END: ${{ vars.MAINTENANCE_END }}
       NYCID_CLIENT_ID: ${{ vars.NYCID_CLIENT_ID }}
       NYCID_DOMAIN: ${{ vars.NYCID_DOMAIN }}
+      ROBOTS_TXT: ${{vars.ROBOTS_TXT}}
```

Comment:
Are we still using `ROBOTS_TXT` as a variable? I can't find where it's read

Reply:
Ah, I will clean that up. Left over from my first attempt.